react-restaurant-review-app/src/components/Login/Login.module.css

```diff
@@ -0,0 +1,95 @@
+.container {
+  display: flex;
+}
+
+form {
+  display: flex;
+  flex-direction: column;
+  height: 700px;
+  width: 500px;
+  border-radius: 10px;
+  border: 0.5px solid var(--gray);
+  background: var(--white);
+  box-shadow: 0px 4px 64px 0px rgba(0, 0, 0, 0.05);
+  padding: 36px;
+  margin-left: auto;
+}
+
+.user-name,
+.password {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: flex-start;
+  gap: 10px;
+  margin-top: 40px;
+}
+
+input {
+  width: 100%;
+  height: 60px;
+  border-radius: 8px;
+  border: 0.6px solid #282828;
+  background: var(--white);
+  padding: 0;
+}
+
+.submitButton {
+  height: 60px;
+  align-items: flex-start;
+  width: 100%;
+  gap: 10px;
+  background: var(--purple);
+  color: white;
+  border-radius: 8px;
+  border: 0.6px solid var(--purple);
+  margin-top: 80px;
+}
+
+h1 {
```

Comment:
All heading styles should be in a global css file since they should affect all components.
In this module there should be only styles that apply to this component and nothing else